{
  "Header": {
    "Original": "CVE-2019-13445 (An issue was discovered in the ROS communications-related packages (ak ...)",
    "Line": 252566,
    "ID": "CVE-2019-13445",
    "Description": "(An issue was discovered in the ROS communications-related packages (ak ...)"
  },
  "Annotations": [
    {
      "Original": "- ros-ros-comm 1.14.3+ds1-11 (bug #947947)",
      "Line": 252567,
      "Type": "package",
      "Package": "ros-ros-comm",
      "Kind": "fixed",
      "Version": "1.14.3+ds1-11",
      "BugNo": 947947
    },
    {
      "Original": "[buster] - ros-ros-comm 1.14.3+ds1-5+deb10u1",
      "Line": 252568,
      "Type": "package",
      "Release": "buster",
      "Package": "ros-ros-comm",
      "Kind": "fixed",
      "Version": "1.14.3+ds1-5+deb10u1"
    },
    {
      "Original": "[stretch] - ros-ros-comm 1.12.6-2+deb9u2",
      "Line": 252569,
      "Type": "package",
      "Release": "stretch",
      "Package": "ros-ros-comm",
      "Kind": "fixed",
      "Version": "1.12.6-2+deb9u2"
    },
    {
      "Original": "NOTE: https://github.com/ros/ros_comm/issues/1738",
      "Line": 252570,
      "Type": "NOTE",
      "Description": "https://github.com/ros/ros_comm/issues/1738"
    },
    {
      "Original": "NOTE: https://github.com/ros/ros_comm/pull/1741",
      "Line": 252571,
      "Type": "NOTE",
      "Description": "https://github.com/ros/ros_comm/pull/1741"
    }
  ]
}